softlayer/resource_softlayer_scale_group.go

```diff
@@ -131,6 +131,31 @@ func resourceSoftLayerScaleGroup() *schema.Resource {
 				Type:     schema.TypeList,
 				Optional: true,
 				Elem:     &schema.Schema{Type: schema.TypeInt},
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					o := strings.TrimRight(strings.TrimLeft(old, " ["), "] ")
```

Comment:
`o` only contains one element of an array. For example, if ```network_vlans_ids = [1001,1002,1003,1004] ``` in tf file and terraform's read function returns `[1004,1003,1002,1001]`, DiffSuppressfunc will be invoked 4 times as follows:
1st call : o = 1001, n = 1004
2nd call : o = 1002, n = 1003
3rd call : o = 1003, n = 1002
4th call : o = 1004, n = 1001